src/components/Listing/ListingElement/ListingElement.tsx

```diff
@@ -22,6 +23,7 @@ type ListingElementProps = {
   imageSrcSet?: string;
   imageWidth?: string;
   hasMetaLabel?: boolean;
+  linkTitle?: string;
```

Comment:
`linkText` is probably a better name than `linkTitle` as this isn't actually related to the title at all

Reply:
@creido-welly I've made changes as you've suggested.